addon/internal-session.js

```diff
@@ -110,7 +128,7 @@ export default Ember.ObjectProxy.extend(Ember.Evented, {
     if (!Ember.isEmpty(this.authenticator)) {
       Ember.set(data, 'authenticated', Ember.merge({ authenticator: this.authenticator }, data.authenticated || {}));
     }
-    this.store.persist(data);
+    return this._callStoreAsync('persist', data);
   },
```

Comment:
Agree, but it does that. The discussion here looks like this change is in `setUnknownProperty` but it's actually in `_updateStore`. I didn't change `setUnknownProperty`